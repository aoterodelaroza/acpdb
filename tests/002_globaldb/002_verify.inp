## check: 002_verify.out -a1e-10
## delete: 002_verify.db
## labels: regression quick

verbose
system rm -f 002_verify.db
connect 002_verify.db
verify

connect ../zz_source/bsip2-small/bsip2.db
verify
