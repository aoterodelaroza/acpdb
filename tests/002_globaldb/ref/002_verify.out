%% verbose
%% system rm -f 002_verify.db
* SYSTEM: rm -f 002_verify.db

%% connect 002_verify.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 002_verify.db
Creating skeleton database 

%% verify
* VERIFY: verify the consistency of the database 

Checking the litrefs in sets are known
Checking the litrefs in methods are known
Checking the structures in properties are known
Checking the number of values and structures in the evaluations table
Checking the number of values and structures in the terms table

%% connect ../zz_source/bsip2-small/bsip2.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file ../zz_source/bsip2-small/bsip2.db
Connected database is sane

%% verify
* VERIFY: verify the consistency of the database 

Checking the litrefs in sets are known
Checking the litrefs in methods are known
Checking the structures in properties are known
Checking the number of values and structures in the evaluations table
Checking the number of values and structures in the terms table

